### PR TITLE
Books pug

### PR DESCRIPTION
**Pull request by eledwinn**
## Summary

1. All templates (.html files) writed with pug

## Screenshot
N/A

## Trello Card
https://trello.com/c/xZ2EWjMl/18-dev-stack-pug

### CODE REVIEW

File: app/book-detail/book-detail.template.pug [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.pad-l-3.pad-r-3.mar-t-3
+  a(ui-sref="booksList")
+    small &lt; Volver
+.main-container
+  .main-body.mar-t-3.book-detail.splitter
+      div
+        img(src="assets/book_logo.png" alt="book image" ng-src="{{$ctrl.book.image_url}}"  class="book-detail-image")
+      .book-detail-content.mar-l-3
+        h1.book-detail-title {{$ctrl.book.title}}
```
Comment (FrankIglesias) [inline app/book-detail/book-detail.template.pug]: use this syntax for texts
```
p.class
  | this is a text
```

File: app/books/books.template.pug [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      items="$ctrl.filters"
+      ng-model="$ctrl.filterField"
+      class="mar-r-3")
+    w-input(
+      placeholder="Buscar..."
+      on-search="$ctrl.search()"
+      ng-model="$ctrl.filterCriteria")
+  .book-items.main-body
+    book-item(book="book" ng-repeat="book in filteredBooks = ($ctrl.books | filter:$ctrl.filter)")
+    p(ng-if="filteredBooks.length === 0") No se han encontrado resultados
```
Comment (FrankIglesias) [inline app/books/books.template.pug]: remove the === 0

File: app/common/w-input/w-input.template.pug [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    type="text"
+    class="w-input"
+    placeholder="{{$ctrl.placeholder}}"
+    ng-model="$ctrl.inputValue"
+    ng-keypress="$ctrl.onKeyPress($event)"
+    ng-change="$ctrl.inputValueChanged()")
+  button(ng-if="!$ctrl.items"
+    class="w-input-button fa fa-search"
+    ng-click="$ctrl.onSearch()")
+
```
Comment (FrankIglesias) [inline app/common/w-input/w-input.template.pug]: remove this line

File: app/index.pug [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    script(src="lib/@uirouter/angularjs/release/angular-ui-router.js")
+    script(src="common/w-input/w-input.component.js")
+    script(src="core/book/book.module.js")
+    script(src="core/book/book.service.js")
+    script(src="book-item/book-item.component.js")
+    script(src="books/books.module.js")
+    script(src="books/books.component.js")
+    script(src="book-detail/book-detail.module.js")
+    script(src="book-detail/book-detail.component.js")
+    script(src="app.js")
```
Comment (FrankIglesias) [inline app/index.pug]: add EOL

File: app/scss/book-detail.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
+.book-detail {
+  display: flex;
+}
+
+.book-detail-image {
+  width: 191px;
```
Comment (FrankIglesias) [inline app/scss/book-detail.scss:7]: why not 190px?

File: app/scss/button.scss [hunk 6]
```diff
@@ -0,0 +1,10 @@
+.btn {
+  padding: 8px;
```
Comment (FrankIglesias) [inline app/scss/button.scss]: 10px?

File: app/scss/subtitle.scss [hunk 7]
```diff
@@ -0,0 +1,5 @@
+.subtitle {
+  color: $green;
+  font-size: 1.3rem;
```
Comment (FrankIglesias) [inline app/scss/subtitle.scss]: use px

File: package.json [hunk 8]
```diff
@@ -7,9 +7,9 @@
     "postinstall": "npm run copy-libs",
     "update-deps": "npm update",
     "postupdate-deps": "npm run copy-libs",
-    "copy-libs": "cpx \"node_modules/{angular,angular-*,html5-boilerplate/dist}/**/*\" app/lib -C",
+    "copy-libs": "cpx \"node_modules/{angular,angular-*,@uirouter/angularjs}/**/*\" app/lib -C",
     "prestart": "npm install",
-    "start": "http-server -a localhost -p 8000 -c-1 ./app",
+    "start": "concurrently --kill-others \"sass --watch app/scss/application.scss:app/application.css\" \"pug app -w\" \"http-server -a localhost -p 8000 -c-1 ./app\"",
```
Comment (FrankIglesias) [inline package.json:12]: mmmm why this kill others?

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: app/scss/book-detail.scss (hunk 5 above)
Comment (eledwinn) [reply app/scss/book-detail.scss:7]: In the styleguide it has 191px.
image

File: package.json (hunk 8 above)
Comment (eledwinn) [reply package.json:12]: With --kill-others switch, all commands are killed if one dies

File: app/book-detail/book-detail.template.pug [hunk 9]
```diff
@@ -0,0 +1,28 @@
+.pad-l-3.pad-r-3.mar-t-3
+  a(ui-sref="booksList")
+    small &lt; Volver
+.main-container
+  .main-body.mar-t-3.book-detail.splitter
+      div
```
Comment (FrankIglesias) [inline app/book-detail/book-detail.template.pug]: I think this div can be deleted

File: app/book-item/book-item.component.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+  module('bookItem', []).
+  component('bookItem', {
+    templateUrl: 'book-item/book-item.template.html',
+    bindings: { 
+      book: '<',
+      small: '@'
+    },
+    controller: function BooksListController() {
+      var ctrl = this;
+
```
Comment (FrankIglesias) [inline app/book-item/book-item.component.js]: remove this line

File: app/core/book/book.service.js [hunk 11]
```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+angular.module('core.book').factory('Books', [function () {
```
Comment (FrankIglesias) [inline app/core/book/book.service.js:3]: check indentation

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**